channeldb/graph.go

```diff
@@ -2479,8 +2475,16 @@ func delEdgeUpdateIndexEntry(edgesBucket kvdb.RwBucket, chanID uint64,
 	return nil
 }
 
-func (c *ChannelGraph) delChannelEdge(edges, edgeIndex, chanIndex, zombieIndex,
-	nodes kvdb.RwBucket, chanID []byte, isZombie, strictZombie bool) error {
```

Comment:
Nice, was thinking why can we just get rid of this  `node` var but was not used 😀